webpack.config.js

```diff
@@ -4,10 +4,12 @@ const webpack = require('webpack');
 // File filter for targeting specific spec files.
 const defaultSpecFileFilter = "/_spec\\.js$/";
 
+const isProduction = process.env.NODE_ENV === "production";
+
 module.exports = {
   entry: "./src/entry.js",
 
-  devtool: 'inline-source-map',
+  devtool: (isProduction ? 'source-map' : 'inline-source-map'),
```

Comment:
Do we want source maps at all for production?

Reply:
`production` here is not the production Rails environment. Production here is how the deliverable for this repo will be generated when publishing the node module. In this case, it says, bundle an external sourcemap file with the JS file in`dist/`. Consumers, namely Web, will choose whether to use the sourcemap. In the case of Web, the soucemap will only be used in test and development Rails environments.

Reply:
Gotcha, thanks for the explanation!